src/network/networkd-wwan-bus.c

```diff
@@ -0,0 +1,1285 @@
+                }
+
+                if (streq_ptr(left, "allowed-auth")) {
+                        MMBearerAllowedAuth auth = prop_auth_lookup(right);
+
+                        r = sd_bus_message_append(m, "{sv}", left, type, (uint32_t)auth);
+                        continue;
+                }
+
+                switch (type[0]) {
+                case SD_BUS_TYPE_BOOLEAN:
+                        r = parse_boolean(right);
+                        if (r < 0)
+                                return -EINVAL;
+                        r = sd_bus_message_append(m, "{sv}", left, type, (bool)r);
+                        break;
+                case SD_BUS_TYPE_INT32:
+                        r = safe_atoi(right, &right_i);
+                        if (r < 0)
+                                return -EINVAL;
+                        r = sd_bus_message_append(m, "{sv}", left, type, (int32_t)right_i);
+                        break;
+                case SD_BUS_TYPE_UINT32:
+                        r = safe_atou(right, &right_u);
+                        if (r < 0)
+                                return -EINVAL;
+                        r = sd_bus_message_append(m, "{sv}", left, type, (uint32_t)right_u);
+                        break;
+                case SD_BUS_TYPE_STRING:
+                        _fallthrough_;
+                default:
+                        r = sd_bus_message_append(m, "{sv}", left, type, right);
+                        break;
+                }
+                if (r < 0)
+                        return bus_log_create_error(r);
+        }
+
+        r = sd_bus_message_close_container(m);
+        if (r < 0)
+                return bus_log_create_error(r);
+
+        return sd_bus_call_async(bus, slot, m, callback, userdata, 0);
+}
+
+static void modem_simple_connect(Modem *modem) {
+        Link *link;
+        int r;
+
+        /* Already have simple connect in progress? */
+        if (modem->slot_connect)
+                return;
+
+        if (modem->reconnect_state != MODEM_RECONNECT_SCHEDULED)
+                return;
+
+        /*
+         * If port name is not known yet then wait for the reconnect
+         * timer to trigger reconnection later on.
+         */
+        if (!modem->port_name)
+                return;
+
+        (void) link_get_by_name(modem->manager, modem->port_name, &link);
+        if (!link) {
+                log_error("ModemManager: cannot find link for %s", modem->port_name);
+                return;
+        }
+
+        /* Check if .network file found at all */
+        if (!link-> network) {
+                log_warning("ModemManager: no .network file provideded for %s", modem->port_name);
+                return;
+        }
+
+        /* Check if we are provided with simple connection properties */
+        if (!link->network->modem_simple_connect_props) {
+                log_warning("ModemManager: no simple connect properties provideded for %s", modem->port_name);
```

Comment:
Also here. Please downgrade. The user may simply not want networkd to connect.

Reply:
will do